docs/source/services/services.rst

```diff
@@ -0,0 +1,256 @@
+        @yaml_tag(eager=True)
+        class ExampleService(object):
+            def __init__(option_1):
+                ...
+            async def run():
+                ...
+
+    .. rubric:: Example configuration
+
+    .. code-block:: yaml
+
+        Services:
+            ExampleService:
+                !ExampleService
+                option_1: my_option_1
+            Service_2:
+                !TardisService2
+                option_123: my_option_123
+
+REST Service
+------------
+
+.. content-tabs:: left-col
+
+    The :py:class:`~tardis.rest.service.RestService` provides a REST API to access the information stored about managed
+    ``Drones`` inside the :py:class:`~tardis.plugins.sqliteregistry.SqliteRegistry` plugin. Currently access is read
+    only and limited to either list all managed resources or to get the state of a particular resource. All supported
+    REST API calls and their parameters are described in REST API documentation available on
+    ``http://<hostname>:<port>/docs`` after starting the service. The REST service is using JSON Web Token (JWT) and
+    OAuth2 scopes for authentication and authorization.
+
+    .. note::
+
+        The REST service currently supports only read access to the
+        :py:class:`~tardis.plugins.sqliteregistry.SqliteRegistry` using the ``resources:get`` OAuth2 scope. However,
+        this could be extended in the future to support also DB updates and deletions using ``resources:put`` and
+        ``resources:delete`` OAuth2 scopes. In addition, plans exists to store the ``TARDIS`` configuration in a
+        database as well and allow to use the REST service to perform configuration updates at run time.
+
+
+    .. warning::
+
+        In order to enable the :py:class:`~tardis.rest.service.RestService`, the
+        :py:class:`~tardis.plugins.sqliteregistry.SqliteRegistry` plugin has to be enabled as well.
+
+
+Available configuration options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. content-tabs:: left-col
+
+    .. table:: General options available to configure the REST service
+
+        +----------------+-------------------------------------------------------------------+-----------------+
+        | Option         | Short Description                                                 | Requirement     |
+        +================+===================================================================+=================+
+        | host           | Hostname or IP the REST Service is listening on                   |  **Required**   |
+        +----------------+-------------------------------------------------------------------+-----------------+
+        | port           | Port the REST Service is listening on                             |  **Required**   |
+        +----------------+-------------------------------------------------------------------+-----------------+
+        | secret_key     | 32 byte secret key to use to generate and validate JWTs           |  **Required**   |
```

Comment:
This might be a bit more readable:

```suggestion
        | secret_key     | 32 byte secret key to generate and validate JWTs           |  **Required**   |
```